src/bika/lims/browser/contact.py

```diff
@@ -78,14 +78,26 @@ def get_users(self):
         users_view = UsersOverviewControlPanel(self.context, self.request)
         return users_view.doSearch("")
 
+    @view.memoize
+    def get_clients_groups(self):
+        """Returns the client-specific groups
+        """
+        groups = []
+        uc = api.get_tool("uid_catalog")
```

Comment:
Please use the `senaite_client_catalog` instead

Reply:
noted

Reply:
Done with https://github.com/senaite/senaite.core/pull/2527/commits/add880883fdde48952e67cbd918f98abd6226f16